src/views/Home/components/NewsCard/index.jsx

```diff
@@ -20,12 +21,25 @@ const NewsCard = () => {
               Today's Student News
             </Grid>
             <Grid item xs={5} align="right">
-              <Button variant="contained" color="secondary" component={Link} to="/news">
+              <Button
+                variant="contained"
+                //color="secondary"
+                style={{
+                  backgroundColor: gordonColors.primary.cyan,
+                  color: gordonColors.neutral.grayShades[50],
+                }}
```

Comment:
The `color=secondary` prop makes this button cyan in a way that is consistent with the rest of the site. Is there a reason you replaced that with these inline styles?

Reply:
If I remember right at this point I was doing a lot in inline styles and saw a few other places using inline-I'll switch it back and keep it as it was